_docs/2.0.0/usage/menas-quick-start.md

```diff
@@ -0,0 +1,79 @@
+
+Firs thing we need to do tobe able to run standardization and conformance is creating of a data schema for the job. Schema represents the data, its types, defaults, etc. If we click on the schema in the navigation pane it will open up all schemas view. This view will be currently empty and at the bottom there is a button `+ New Schema`. Pressing the button will lead to modal window showing up.
+
+{% include image.html name="usage/menas_schemas_empty.png" alt="Menas Empty Schemas Pane" %}
+
+Creation of a new schema has only `Name` and `Description`. Names in all Menas models must always be unique and a combination of Name and Version will always provide a unique identifier of an object.
+
+{% include image.html name="usage/menas_new_schema.png" alt="Menas New Schema" %}
+
+Clicking on a Save button, Menas will validate the Name inputed and save the new object.
+
+{% include image.html name="usage/menas_newly_created_schema.png" alt="Menas Newly Created Schema" %}
+
+Now we need to input a schema. Schemas can be uploaded as many types. Spark Struct, Cobol Copybook or Avro Schema. Choose what you like the most and upload your schema. For test purposes here, we will use a simple Spart Struct schema:
+
+```json
+{
+  "type" : "struct",
+  "fields" : [ {
+    "name" : "A1",
+    "type" : "string",
+    "nullable" : true,
+    "metadata" : { }
+  }, {
+    "name" : "A2",
+    "type" : "string",
+    "nullable" : true,
+    "metadata" : { }
+  }, {
+    "name" : "A3",
+    "type" : "string",
+    "nullable" : true,
+    "metadata" : { }
+  } ]
+}
+```
+
+For more rules about schemas check the [this][schema] schema section. Now click the tab `Upload New`.
+
+{% include image.html name="usage/menas_upload_schema.png" alt="Menas Upload Schema" %}
+
+Click button `Browse...` to select schema file from your local disk. When selected click upload. This will validate inputed schema and if everything is allright, it will upload the schema. You will be moved back to `Basic Info` tab and you can see that your Version rose by 1. If you click the tab `Fields` you will se your new schema.
+
+{% include image.html name="usage/menas_schema_view.png" alt="Menas Schema View" %}
+
+Moving on to creating a data set. Data set in Menas represents your a general location where your data sets of the same type will be periodically landed. You will have to specify to which schema the data comply to, where they are and where they should be when standardization and conformance are finished.
+
+Creating new Dataset goes the same way as the schema. We open the left menu pane and select Dataset and then click `+ New dataset`.
+
+We will fill in Name and Description same as in Schema, then we will have to asign schema. When you click on select schema, find the schema you created and select it. It will automaticaly select highest version, but in a combobox bellow you can choose which schema version you would like to use.
```

Comment:
```suggestion
We will fill in Name and Description same as in Schema, then we will have to assign a schema. When you click on _Select schema_, find the schema you created and select it. It will automatically select the highest version, but in a combobox bellow you can choose which schema version you would like to use.
```